webapp/scripts/grass.js

```diff
@@ -243,7 +243,7 @@ function listVector(mapset) {
  * @return {string[]} names of available maps
  */
 function listUserVector() {
-  return grass('PERMANENT', 'g.list type=vector mapset=*').trim().split('\n').filter(filterDefaultLayers)
+  return grass('PERMANENT', 'g.list -m type=vector').trim().split('\n').filter(filterDefaultLayers)
```

Comment:
Why do you print fully-qualified map names, only to remove the mapset part later?

Reply:
Because if you remove '-m', some of the results are still fully-qualified map names, so the result is more consistent and manageable with the '-m'. I highly recommend you try out the alternatives yourself during the review. I cannot create comments for every line I change.

Reply:
> Because if you remove '-m', some of the results are still fully-qualified map names, so the result is more consistent and manageable with the '-m'. 
I see.
> I highly recommend you try out the alternatives yourself during the review. I cannot create comments for every line I change.
All I was asking was an explanation like the above. I cannot checkout your code and evaluate the effects of different variations on each line of code either.
After all, the outcome of the review process should be well-documented code, and in this case I thought a comment was helpful.

Reply:
Will add the comments. Thanks